services/QuillLMS/bin/dev/airbyte

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env ruby
```

Comment:
Is this file supposed to be in here?

Reply:
Yes. This was pulled directly from `develop`, so I could work on airbyte without checking out a different branch. I should have explained that.